lib/ansible/parsing/dataloader.py

```diff
@@ -72,14 +73,14 @@ def __init__(self):
         self.set_vault_secrets(None)
 
     # TODO: since we can query vault_secrets late, we could provide this to DataLoader init
-    def set_vault_secrets(self, vault_secrets):
+    def set_vault_secrets(self, vault_secrets: list[str]):
```

Comment:
While you are here, can you please fix typos on https://github.com/ansible/ansible/blob/6db006a5fa18b710e5ad0bc90e5ea5494a6b68c2/lib/ansible/parsing/dataloader.py#L48 and https://github.com/ansible/ansible/blob/6db006a5fa18b710e5ad0bc90e5ea5494a6b68c2/lib/ansible/parsing/dataloader.py#L69. Instead of `self.set_vault_password(` it should read `self.set_vault_secrets(`

Reply:
I updated the docstring and removed the code comment.